web/admin/application/configs/klear/klear.yaml

```diff
@@ -289,125 +305,141 @@ production:
           class: ui-silk-lightning-go
           description: _("List of %s", ngettext('DDI', 'DDIs', 0))
           showOnlyIf: ${auth.companyRetail}
-        OutgoingDDIRulesList:
-          title: ngettext('Outgoing DDI Rule', 'Outgoing DDI Rules', 0)
-          class: ui-silk-script-lightning
-          description: _("List of %s", ngettext('Outgoing DDI Rule', 'Outgoing DDI Rules', 0))
-          showOnlyIf: ${auth.companyVPBX}
-        ExternalCallFiltersList:
-          title: ngettext('External call filter', 'External call filters', 0)
-          class: ui-silk-arrow-branch
-          description: _("List of %s", ngettext('External call filter', 'External call filters', 0))
+        RoutingObjects:
+          title: _("Routing objects")
+          class: ui-silk-arrow-switch
           showOnlyIf: ${auth.companyVPBX}
+          submenus:
+            FriendsList:
```

Comment:
I think the most used items in this menu are:

- IVRs
- HuntGroups
- Queues
- Conditional Routes
- Friends
- Conferences
- Faxes

Also, the label _Routing_ could be enough (without the _object_ word): Routing and Routing tools